src/app/backend/resource/customresourcedefinition/objects.go

```diff
@@ -31,6 +31,9 @@ import (
 type CustomResourceObject struct {
 	TypeMeta   metav1.TypeMeta   `json:"typeMeta"`
 	ObjectMeta metav1.ObjectMeta `json:"objectMeta"`
+
+	// Represents whether or not the CRD's 'scale' subresource is enabled.
+	IsScalable bool `json:"isScalable"`
```

Comment:
s/IsScalable/Scalable

I'd change that everywhere. `scalable` itself already suggests that it is a boolean value.

Also maybe instead of adding a new `scalable` information only to CRD, we could maybe extend `TypeMeta` information with `scalable` variable. This would simplify frontend part a lot.

@maciaszczykm WDYT?

Reply:
Fine for me, extending metadata is always good when there is opportunity (all resources have the same property).

Reply:
I agree that extending `TypeMeta` simplifies things. I didn't do that originally because I thought the fields in `TypeMeta` should always correspond to the main k8s representation.

Reply:
They do not have to be the same. This is our internal API used by the frontend. If it is useful for us, then it can be added.

Reply:
I'll work on extending the field today.

Reply:
Done. Should I move [this implementation](https://github.com/kubernetes/dashboard/blob/master/src/app/frontend/common/components/list/column/menu/component.ts#L34) of `scalableResources` for other resources to the backend now?